packages/interactivity/src/directives.js

```diff
@@ -267,15 +267,71 @@ export default () => {
 	// data-wp-watch--[name]
 	directive( 'watch', ( { directives: { watch }, evaluate } ) => {
 		watch.forEach( ( entry ) => {
-			useWatch( () => evaluate( entry ) );
+			useWatch( () => {
+				if ( SCRIPT_DEBUG ) {
+					// eslint-disable-next-line no-unused-vars
+					const start = performance.now();
```

Comment:
I think ESLint is trying to point out an actual error here. `const` is scope-based, which means `start` is only defined inside the `if` condition. It would be safer to use `let start` in the outer scope. I'm wondering if that works correctly after transpilation as that depends on what to happens with `const` here and the surrounding `if` statement.

Reply:
> I think ESLint is trying to point out an actual error here

@gziolo do you see an ESLint warning?